### ISSUE TITLE
Downloading Pyne on ubuntu 14.04

### ISSUE THREAD

**Issue by KennellyT** [pyne/pyne#1029]
Hey,
I am Tyler Kennelly and I am trying to download pyne onto ubuntu 14.04.
I have tried source build but it is having issues with recognizing Cmake (Figure 1)
I have also tried to gather it from conda install but it does not recognize the version number 0.5.11
I tried to gather the developer version as well but the conda-recipes-master does not take the input "pyne". (Figure 2)

Has anyone else seen these issues and can offer advice? 
Thank you
![screenshot from 2018-05-05 18 00 16](https://user-images.githubusercontent.com/31637359/39668372-8277603e-5091-11e8-94f4-f4100bfe17b5.png)
![screenshot from 2018-05-05 18 14 36](https://user-images.githubusercontent.com/31637359/39668373-8560e16c-5091-11e8-8aec-bc4a1a2b5edd.png)

**Comment by katyhuff** [comment]
In the first frame, it is recognizing CMake. CMake appears to be running, but then throws an error during the build. The error will be in the CMakeError.log file referenced in the error.

I'm not sure what's up with the conda build.

**Comment by KennellyT** [comment]
Thank you Dr. Huff, I will check the CMakeError.log file for its information. 
Thank you

**Comment by katyhuff** [comment]
Hi @KennellyT , can you please summarize what you determined was the source of this issue (for posterity? https://xkcd.com/979/ ) 

![https://imgs.xkcd.com/comics/wisdom_of_the_ancients.png](https://imgs.xkcd.com/comics/wisdom_of_the_ancients.png)

**Comment by KennellyT** [comment]
It seems that the main issue(s) I have had with pyne came from where I downloaded PYNE from. For example, when I downloaded PYNE as:
conda install -c cyclus -c pyne pyne=VERSION

Issues such as utils.py not having use_end_ftod in utils.py, or nuc_data_make wouldn't be recognized. 

When I found and installed PYNE via conda-forge:
conda install -c conda-forge -c pyne pyne=VERSION

The download was successful with no errors and I was able to use PYNE right after the download. I would guess it would have to do with the differences between Pyne via the cyclus channel and Pyne via conda-forge. 
See:[Downloading Pyne #1012](https://github.com/pyne/pyne/issues/1012)
[change conda channel #1015](https://github.com/pyne/pyne/pull/1015)
